src/Tizen.NUI/src/public/WindowSystem/TzShell.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Tizen.Applications.Exceptions;
+
+namespace Tizen.NUI
+{
+    namespace WindowSystem
+    {
+        namespace WsShell
+        {
+            /// <summary>
+            /// Tizen Window System Shell
+            /// </summary>
+            public class TzShell : IDisposable
```

Comment:
DO NOT use any acronyms. Please rename TzShell.

Reply:
I changed it to TizenShell